plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version4/Customers/class-wc-rest-customers-v4-controller-tests.php

```
@@ -949,8 +951,312 @@ public function test_comprehensive_value_validation(): void {
+		$this->assertCount( 2, $response_data );
+		// Customer1 should come first (higher total), then customer2 (lower total).
+		$this->assertEquals( $response_data[0]['id'], $customer1->get_id() );
+		$this->assertEquals( $response_data[1]['id'], $customer2->get_id() );
+	}
+
+	/**
+	 * Test orderBy with last_active sorting.
+	 */
+	public function test_orderby_last_active(): void {
+		global $wpdb;
+		$site_specific_key = rtrim( $wpdb->get_blog_prefix( get_current_blog_id() ), '_' );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$site_specific_key'. (undefined)

(UnusedLocalVariable)